src/main/resources/application-email-addresses.yaml

```diff
@@ -28,9 +28,10 @@ hearingCentreTelephoneNumber:
   hattonCross: ${IA_HEARING_CENTRE_HATTON_CROSS_TELEPHONE:0300 123 1711}
   glasgow: ${IA_HEARING_CENTRE_GLASGOW_TELEPHONE:0300 123 1711}
 
-endAppealHomeOfficeEmailAddress: ${IA_HOME_OFFICE_END_APPEAL_EMAIL:ho-end-appeal@example.com}
-allowedAppealHomeOfficeEmailAddress: ${IA_HOME_OFFICE_ALLOWED_APPEAL_EMAIL:ho-allowed-appeal@example.com}
-dismissedAppealHomeOfficeEmailAddress: ${IA_HOME_OFFICE_DISMISSED_APPEAL_EMAIL:ho-dismissed-appeal@example.com}
+endAppealHomeOfficeEmailAddress: ${IA_HOME_OFFICE_END_APPEAL_EMAIL:ReformpbAPC@example.com}
```

Comment:
@tarun-palisetty - I think we need to keep the email addresses different to what's in production. I had this discussion with Ravi last week and we effected these changes in ticket RIA-2366. The idea is that it's not easy for anyone to guess what the actual email addresses would be production, simply by looking at the code (publicly visible) within GitHub.

Reply:
modified the email accordingly